index.html

```diff
@@ -5,13 +5,14 @@
   <title>Mobile Version</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel='stylesheet' type='text/css' href='styles.css'>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
     <nav  class="nav-bar display-flex justify-content">
       <div class="name">
-        <a href="#">Francis Ugorji</a>
+        <a href="#" class="style">Francis Ugorji</a>
       </div>
       <div class="icon-menu">
         <img class="icon-menu" src="Images/Icon-Menu.png" alt="A Menu Bar Icon">
```

Comment:
- [ ] it should be a section tag that nest a header tag
![fff](https://user-images.githubusercontent.com/61562626/129428571-cac0c6ab-f611-4b55-8f26-edbe280531eb.jpeg)

Reply:
In the first project that had the header-headline-branch, we were required to use a nav tag instead of a section tag for the logo and the menu image. 

In this branch, I nested the <header> in a <section> as seen in the HTML file from line 95-99.

Thank you for the constructive feedback.